test_runner/regress/test_import_pgdata.py

```diff
@@ -340,6 +341,225 @@ def validate_vanilla_equivalence(ep):
         br_initdb_endpoint.safe_psql("select * from othertable")
 
 
+@pytest.mark.timeout(10 * 60)
```

Comment:
If this really is a long running test we should either:
* make it release build only via `@skip_in_debug_build`
* move it to the performance suite `test_runner/performance` where it does not run in parallel with other tests.

Reply:
Although, I don't get which part of the test could take this long. It doesn't shuffle much data.

Reply:
We are polling for status (in both mentioned tests), I have already seen it running whole 45-60 minutes in CI because of something not happening, don't want to have this again so limiting whole test. Can consider introducing timeout into polling specifically, but idk, this seemed like an easy solution. Can add a comment there

Reply:
Well if the expectation is that the test is quick and there's cases when it's not, then something is wrong. Worth investigating. We really don't want long running tests in the normal regression suite.

Reply:
I'm confused. IIRC the default timeout is 5 minutes, so you're increasing the timeout to 10 minutes here. But at the same time you're saying that it could take 45-60 minutes in CI. That means we'll see this failing in the CI. Are you trying to:

- make the test fail earlier, if it gets stuck for 45-60 minutes, or
- make the test pass, if it takes somewhere between 5-10 minutes?

In either case, I think we need to fix the test so that it reliably runs in < 1 minute or so. Long-running tests are painful, and flaky tests even more so.

Reply:
Is there a default timeout of 5 minutes? I did not know then, then I can remove this one. I remember hitting all-ci-run-timeout once, might be because we are close to it and a few 5min timeouts indeed reached overall timeout.

It is reliably runs under 1 minute, but there is polling of the timeline status so if for some reason it did not move through, then I will hit some timeout.

Reply:
removed timeouts, default are enough

Reply:
Removed these lines, default 5 min is enough